ui/core/src/model/time.ts

```diff
@@ -167,6 +167,36 @@ export function formatDuration(duration: Duration): DurationString {
   return result.join('') as DurationString;
 }
 
+export function formatLongDuration(duration: Duration): DurationString {
```

Comment:
There is no native helpers for this? 🤔

Reply:
ah yes ! https://date-fns.org/v3.6.0/docs/formatDuration
Okay I ill use it then :D

Reply:
This formatDuration works like a charm 
![image](https://github.com/perses/perses/assets/10258608/c643b900-dc04-4cd2-8f39-c1741c739ab1)